README.md

```diff
@@ -1 +1,48 @@
-# another-todo-app
\ No newline at end of file
+# Do it List App ~ JavaScript
+
+> A Todo list built using webpack and served by a webpack dev server.
+
+# Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+# Demo Link
+Visit [Link](https://fr3elancer.github.io/todo-app/)
+
+# Getting started
+
+## To get a local copy of this repository kindly follow the steps below.
+- Scroll to top of this current repository
+- Click on the `Code` button with background color green on the right end corner
+- Click on the clipboard icon on the extreme right of the dropdown to copy the repository link
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Type `git clone (copied link)` on the currently opened terminal or command prompt
+
+# Getting Started 
+
+- Open your Terminal and Navigate to the project destination and type `npm install`
```

Comment:
Kindly complete the steps that would take a user to run your app locally.

running only `npm install` won't start the app.

Kindly write the correct steps and fix your running commands like `npm run server`

Reply:
✔️